Answers.md

```diff
@@ -0,0 +1,4 @@
+1. Preprocessing is basically taking a complicated language and creating a simpler way of writing code and having a program basically translate a more human readable material to the standard format that a computer can read.
```

Comment:
Preprocressing adds extra functionality to CSS that's found in other programming languages: _variables_, _operatioons_ and _functions_ to name a few.